### PR TITLE
script to bump copyright year

### PR DESCRIPTION
**Pull request by solarw**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes and CI passes too
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

DELETE INCLUSIVE THIS AND BELOW FOR STANDARD PR
------

## Release summary

Version number: [e.g. 1.0.1]

## Release details

Describe in short the main changes with the new release.

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side), from `develop`
- [ ] Lint and unit tests pass locally and in CI
- [ ] I have checked the fingerprint hashes are correct by running (`scripts/generate_ipfs_hashes.py`)
- [ ] I have regenerated the latest API docs
- [ ] I built the documentation and updated it with the latest changes
- [ ] I have added an item in `HISTORY.md` for this release
- [ ] I bumped the version number in the `aea/__version__.py` file.
- [ ] I bumped the version number in every Docker image of the repo and published it. Also, I built and published them with tag `latest`  
      (check the READMEs of [`aea-develop`](../develop-image/README.md#publish) 
      and [`aea-user`](../user-image/README.md#publish))
- [ ] I have pushed the latest packages to the registry.
- [ ] I have uploaded the latest `aea` to PyPI.
- [ ] I have uploaded the latest plugins to PyPI.

## Further comments

Write here any other comment about the release, if any.

### CODE REVIEW

File: .github/workflows/workflow.yml [hunk 1]
```diff
@@ -154,6 +154,11 @@ jobs:
         run: tox -e check_generate_all_protocols
       - name: Generate Documentation
         run: tox -e docs
+      - name: Check copyright year is up to date
+        run: |
+          ./scripts/bump_year.sh $(date +%Y)
+          git diff --quiet||(echo "Working tree is dirty. Run ./scripts/bump_year.sh!"; exit 1)
```
Comment (5A11) [inline .github/workflows/workflow.yml:160]:
```suggestion
          git diff --quiet||(echo "Some files have the incorrect year in their copyright header. Run ./scripts/bump_year.sh!"; exit 1)
```

File: aea/cli/utils/config.py [hunk 2]
```diff
@@ -19,7 +19,7 @@
 # -*- coding: utf-8 -*-
 # ------------------------------------------------------------------------------
 #
-#   Copyright 2018-2020 Fetch.AI Limited
```
Comment (5A11) [inline aea/cli/utils/config.py:22]: Why does this file have two headers? Please delete if it's a mistake.

Comment (solarw) [reply aea/cli/utils/config.py:22]: done

**Review by 5A11 — CHANGES_REQUESTED** [review]
Please revert the changes on all protocol under `packages/...`.
For protocols:
- change protocol generator so it does NOT insert a copyright header for a generated protocols. 
- Modify the script that checks protocols' consistency: have it use the generator to generate the protocols but then insert the header manually at the top, then check the protocol is identical (this script is used in CI so should apply to all protocols every time CI process is triggered).

**Comment by solarw** [comment]
updated, no copyright header added with aea protocol generator
fetch.ai protocols updated with copyright header with `./scripts/generate_all_protocols.py`
reviewed

**Review by jrriehl — APPROVED** [review]
LGTM

**Review by 5A11 — CHANGES_REQUESTED** [review]
Please revert changing the `*_pb2` files.

**Review by 5A11 — APPROVED** [review]
lgtm

**Pull request merged**